test/lib/yggdrasil/command/parser_test.exs

```diff
@@ -0,0 +1,17 @@
+defmodule ParserTest do
+  use ExUnit.Case
+  alias Yggdrasil.Command.Parser
+
+  test "parse/1 returns a tagged tuple when parsing is successful" do
+    assert {:ok, _cmd} = Parser.parse "move east"
+  end
+
+  test "parse/1 sets the fields on the struct based on the placeholder" do
+    assert {:ok, cmd} = Parser.parse "move east"
+    assert "east" = cmd.exit
+  end
+
+  test "parse/1 returns :error with reason when parsing is unsuccessful" do
+    assert {:error, _reason} = Parser.parse "IHopeOneDay ThisIsn'tACommand"
```

Comment:
Just to follow up, I made the changes in the `ParseRule` to down case the command pattern string. Then I updated the generated `parse/1` function to down case the input string.